x-pack/plugins/secops/public/containers/overview/overview_host/index.tsx

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { getOr } from 'lodash/fp';
+import React from 'react';
+import { Query } from 'react-apollo';
+import { pure } from 'recompose';
+
+import { GetOverviewHostQuery, OverviewHostData } from '../../../graphql/types';
+import { createFilter } from '../../helpers';
+import { QueryTemplateProps } from '../../query_template';
+
+import { overviewHostQuery } from './index.gql_query';
+
+export interface OverviewHostArgs {
+  id: string;
+  overviewHost: OverviewHostData;
+}
+
+export interface OverviewHostProps extends QueryTemplateProps {
+  children: (args: OverviewHostArgs) => React.ReactNode;
+  sourceId: string;
+  endDate: number;
+  poll: number;
+  startDate: number;
+}
+
+export const OverviewHostQuery = pure<OverviewHostProps>(
+  ({ id = 'overviewHostQuery', children, filterQuery, sourceId, startDate, endDate }) => (
+    <Query<GetOverviewHostQuery.Query, GetOverviewHostQuery.Variables>
+      query={overviewHostQuery}
+      fetchPolicy="cache-and-network"
```

Comment:
I would change this to use `getDefaultFetchPolicy()` unless you need to deviate from using it and we can set the policy there to control most widgets/pages to be the 99% rule.